.github/workflows/code-quality.yaml

```diff
@@ -0,0 +1,47 @@
+name: Code Quality
+
+on:
+    push:
+    pull_request:
+        branches: [ master ]
+    workflow_dispatch:
+
+jobs:
+    php-cs-fixer:
+        runs-on: ubuntu-latest
+        name: Coding Standards
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v3
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '8.1'
```

Comment:
@HypeMC Maybe run this step on `8.2` with `PHP_CS_FIXER_IGNORE_ENV=1` for consistency?

Reply:
Same as the above.